app/code/Magento/Cms/Controller/Adminhtml/Page/Index.php

```diff
@@ -26,16 +28,24 @@ class Index extends \Magento\Backend\App\Action implements HttpGetActionInterfac
      */
     protected $resultPageFactory;
 
+    /**
+     * @var DataPersistorInterface
+     */
+    private $dataPersistor;
+
     /**
      * @param Context $context
      * @param PageFactory $resultPageFactory
+     * @param DataPersistorInterface $dataPersistor
      */
     public function __construct(
         Context $context,
-        PageFactory $resultPageFactory
+        PageFactory $resultPageFactory,
+        DataPersistorInterface $dataPersistor
```

Comment:
```suggestion
        DataPersistorInterface $dataPersistor = null
```
To make it an optional backward compatible argument

Reply:
Thanks @atishgoswami 
I have updated the code as per the suggestion.